revista-ladinia.csl

```diff
@@ -0,0 +1,154 @@
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="anonymous">N.N.</term>
+      <term name="volume">vol.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" delimiter="/" form="short">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <et-al term="and others"/>
+      <substitute>
+        <names variable="editor"/>
+        <text term="anonymous"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-bibliography">
+    <names variable="author">
+      <name name-as-sort-order="all" delimiter="/">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <et-al/>
+      <substitute>
+        <names variable="editor translator">
+          <name name-as-sort-order="all" delimiter="/">
+            <name-part name="family" font-variant="small-caps"/>
+          </name>
+          <label prefix=" (" suffix=")"/>
+          <et-al/>
+        </names>
+        <text term="anonymous"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="issued-year">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher-place">
+    <choose>
+      <if variable="publisher-place">
+        <text variable="publisher-place" suffix=" "/>
+      </if>
+      <else-if type="book" match="any">
+        <text term="no-place" suffix=" "/>
+      </else-if>
+    </choose>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" initialize-with="." disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" collapse="year">
+    <sort>
+      <key macro="author"/>
+      <key macro="issued-year"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter=", ">
+      <group delimiter=" ">
+        <text macro="author"/>
+        <text macro="issued-year"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-min="4" et-al-use-first="1">
+    <sort>
+      <key macro="author-bibliography"/>
+      <key macro="issued-year"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <text macro="author-bibliography" suffix=": "/>
+      <group delimiter=", ">
+        <text variable="title" font-style="italic"/>
+        <group delimiter=": ">
+          <text term="in"/>
+          <text variable="container-title"/>
+        </group>
+        <group delimiter=" ">
+          <text term="volume"/>
```

Comment:
I think you'll want form="short" here, right?

Reply:
right, thanks!